### ISSUE TITLE
[Bug]: create_frame_tops.csh can generate invalid date string in XML filename

### ISSUE THREAD

**Issue by ericlindsey** [gmtsar/gmtsar#113]
When running create_frame_tops.csh, there is an edge case where an image time very close to the next minute (e.g. 59.99 seconds) can get rounded up to 60, resulting in an invalid filename for one of the subswaths. For example, I have the cropped image folder:

    S1A_IW_SLC__1SDV_20181024T131059_20181024T131113_024278_02A83E_A025.SAFE/

Inside here, there is the file: 
    
   s1a-iw3-slc-vv-20181024t131060-20181024t131111-024278-02a83e-006.xml

You can see that the start time in the SAFE folder is 59 seconds, but in the 3rd subswath it rounds to 60. Looking inside, I find the actual start time: 

    <startTime>2018-10-24T13:10:59.993836</startTime>

There must be some rounding step in creating this filename that caused this error. This causes a downstream bug in my automated processing when trying to parse the date from the xml filename to figure out which orbit file to use. 

GMTSAR version 6.0, installed via MacOS Homebrew.

**Comment by Xiaohua-Eric-Xu** [comment]
Yes. These are the two lines where it is doing the rounding I suppose.
```
100   set t1 = `grep startTime new.xml | awk -F">" '{print $2}' | awk -F"<" '{print substr($1,12,2)substr($1,15,2)substr($1,18,9)}' | awk '{printf("%.6d",$1+0.5)}'`
101   set t2 = `grep stopTime new.xml | awk -F">" '{print $2}' | awk -F"<" '{print substr($1,12,2)substr($1,15,2)substr($1,18,9)}' | awk '{printf("%.6d",$1+0.5)}'`
```
Do you want me to change it to `$1` without the `+0.5` ?

**Comment by Xiaohua-Eric-Xu** [comment]
Just made a quick fix here https://github.com/gmtsar/gmtsar/commit/c461c3ad3720fc495b2a6888013081d633d9fcfa Do you mind give it another try?

**Comment by ericlindsey** [comment]
That looks great. I will try it tomorrow and let you know if there is any issue. Thanks!
